src/chip-input/ux-chip-input.ts

```diff
@@ -0,0 +1,190 @@
+import { customElement, bindable, ViewResources, View, processAttributes } from 'aurelia-templating';
+import { DOM } from 'aurelia-pal';
+import { bindingMode } from 'aurelia-binding';
+import { inject } from 'aurelia-dependency-injection';
+import { StyleEngine } from '../styles/style-engine';
+import { Themable } from '../styles/themable';
+import { processDesignAttributes } from '../designs/design-attributes';
+
+@inject(Element, ViewResources, StyleEngine)
+@customElement('ux-chip-input')
+@processAttributes(processDesignAttributes)
+
+export class UxChipInput implements Themable {
+  @bindable public disabled: any = false;
+  @bindable public readonly: any = false;
+  @bindable public theme = null;
+  @bindable public type: any;
+  @bindable public separator: string;
+
+  @bindable({ defaultBindingMode: bindingMode.twoWay })
+  public value: any = undefined;
+
+  @bindable({ defaultBindingMode: bindingMode.twoWay })
+  public chipArray: Array<string> = new Array<string>();
```

Comment:
Hmm, looking back on it I do not particularly like that either. I did like the idea of just calling it chips. This has been changed.